README.md

```diff
@@ -265,7 +265,14 @@ Nope. Search UI is "headless".
 
 You can write support for it into any JavaScript framework. You can even use vanilla JavaScript.
 
-[Read the Headless Core Guide](./ADVANCED.md#headless-core) for more information.
+[Read the Headless Core Guide](./ADVANCED.md#headless-core) for more information, or check
+out the [Vue.js Example](https://github.com/yakhinvadim/vue-search-ui-demo).
```

Comment:
We might want to re-create this as an Elastic repository.

Reply:
Just curious - any specific reason for the manual newline vs. just letting IDEs wordwrap naturally?

Reply:
Oops, no reason :) Thx for noticing.